libraries/crypto/Cargo.toml

```diff
@@ -25,3 +25,4 @@ zeroize = { version = "1.5.7", features = ["derive"] }
 [features]
 std = ["hex", "ring", "untrusted", "serde", "serde_json", "regex", "rand_core/getrandom"]
 with_ctap1 = []
+acvp = []
```

Comment:
I don't think we need a feature. Just make anything public without the feature as necessary!

Reply:
In both crates?

Reply:
Yes, go ahead! It is just accessors that, if useful for us, might be useful for others.

Reply:
Done